Sources/Rendering/OpenGL/ImageResliceMapper/index.js

```diff
@@ -1067,6 +1067,53 @@ function vtkOpenGLImageResliceMapper(publicAPI, model) {
     return [false, 2];
   }
 
+  function isPlaneAxisAligned(normal, imageData) {
+    const d9 = imageData.getDirection();
+    const [orthoSlicing, orthoAxis] = isVectorAxisAligned(normal);
+    return [vtkMath.isIdentity3x3(d9) && orthoSlicing, orthoAxis];
+  }
+
+  function imageToCubePolyData(image) {
+    // First create a cube polydata in the index-space of the image.
+    const cube = vtkCubeSource.newInstance();
```

Comment:
Small memory-related improvement: I think we can create a vtkCubeSource outside of `imageToCubePolyData()`. That way we don't create a new cube source object every time we reslice. vtkCubeSource will generate new polydata whenever its parameters change, so that's unavoidable.